notebooks.go

```diff
@@ -533,6 +534,11 @@ func (s *server) NewNotebook(w http.ResponseWriter, r *http.Request) {
 		notebook.Spec.Template.Spec.Containers[0].Resources.Limits[corev1.ResourceMemory] = req.MemoryLimit
 	}
 
+	// There's more than just the json
```

Comment:
What do you mean that there's more than just the json?

Reply:
This is the wrong comment. Fixed